src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.10.td.json

```diff
@@ -79,11 +86,11 @@
             "FirstNameValue"
           ],
           "caseNameHmctsInternal": "Name Internal",
-          "baseLocationId": "1",
+          "baseLocationId": "${[scenarioContext][testData][request][body][searchCriteria][caseManagementBaseLocationIds][0]}",
           "baseLocationName": "[[ANY_NULLABLE]]",
           "caseManagementCategoryId": "987",
           "caseManagementCategoryName": "Category label",
-          "regionId": "3",
+          "regionId": "${[scenarioContext][testData][request][body][searchCriteria][caseManagementRegionIds][0]}",
```

Comment:
Same as above.

Reply:
resolved in fb506344bf94aaebeabf6903aab21df062468b0d